### PR TITLE
Crs toolchain fixes

### PR DESCRIPTION
**Pull request by RedXanadu**
PR brings the new crs-toolchain pages and changes in line with the existing pages. Corrects minor errors.

### CODE REVIEW

**Comment by RedXanadu** [comment]
@theseion Thanks for sorting all of the crs-toolchain documentation changes and updates.

I've only just had the chance to read the new content. I've made a few minor changes: are you happy with these?

In particular: does line 20 mean what you intended it to mean?

> Provided that the Go binaries are present…

I assumed the original sentence was meant to say "…are present on your machine…".

File: content/development/crs_toolchain.md [hunk 1]
```diff
@@ -1,43 +1,46 @@
 ---
 title: crs-toolchain
-weight: 10
+weight: 12
 disableToc: false
 chapter: false
 ---
 
-> The crs-toolchain is the utility belt of CRS developers. It provides a single point of entry and a consistent interface for a range of different tools. Its core functionality (owed to the great [rassemble-go](https://github.com/itchyny/rassemble-go) (which is itself based on the brain-melting [Regexp::Assemble](https://github.com/ronsavage/Regexp-Assemble) Perl module) is to assemble individual parts of a regular expression into a single expression (with some optimizations).
+> The crs-toolchain is the utility belt of CRS developers. It provides a single point of entry and a consistent interface for a range of different tools. Its core functionality (owed to the great [rassemble-go](https://github.com/itchyny/rassemble-go), which is itself based on the brain-melting [Regexp::Assemble](https://github.com/ronsavage/Regexp-Assemble) Perl module) is to assemble individual parts of a regular expression into a single expression (with some optimizations).
```
Comment (theseion) [inline content/development/crs_toolchain.md:8]: The comma replaced an opening parenthesis, but the closing one is still there.

File: content/development/crs_toolchain.md [hunk 2]
```diff
@@ -1,43 +1,46 @@
...
-If you have a current Go environment, you can simply run
+
+If a current Go environment is present, simply run
 
 ```bash
 go install github.com/coreruleset/crs-toolchain@latest
 ```
 
-Provided that you have the Go binaries on your, you can now run the toolchain from anywhere with
+Provided that the Go binaries are present, the toolchain can now be run from anywhere with
```
Comment (theseion) [inline content/development/crs_toolchain.md:20]: I wanted to write "on the path", i.e. `PATH`.

File: content/development/crs_toolchain.md (hunk 1 above)
Comment (RedXanadu) [reply content/development/crs_toolchain.md:8]: There was one extra opening bracket before. I think the amendment balances out the brackets? But I hope I haven't accidentally changed the meaning of what you wrote.

Removing the hyperlink syntax to make it easier to read, we have:

> Its core functionality (owed to the great rassemble-go, which is itself based on the brain-melting Regexp::Assemble Perl module) is to assemble individual parts of a regular expression into a single expression (with some optimizations).

I think that's correct. But again, I hope it hasn't altered your original meaning.

Comment (theseion) [reply content/development/crs_toolchain.md:8]: Oh, yeah, sorry. that opening parenthesis was actually wrong.

**Pull request merged**